src/go/k8s/tests/e2e/scale-up/01-one-replica.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: cluster-sample
+  labels:
```

Comment:
Because https://github.com/vectorizedio/redpanda/pull/563 and https://github.com/vectorizedio/redpanda/pull/548 are merged maybe we can skip all together those labels?
Maybe add different set of labels?

Reply:
Should we assert the set lebels in `01-assert.yaml` file?

Reply:
yup, good idea, I'll change these